.golangci.yml

```diff
@@ -49,11 +49,34 @@ linters-settings:
   maligned:
     suggest-new: true
   depguard:
```

Comment:
depguard v2 is used in this new linter version and config is totally different

Reply:
we (CLI, SDKs) decided to disabled the linter since we weren't really using it or had any concerns on a wrong import

Reply:
`depguard` we removed it from the list of enabled linters

Reply:
you can run `golangci-lint linters` in the project folder to see a list of enabled/disabled linters

Reply:
ok, i'll do. I see some value, e.g. we're using github.com/go-test/deep our of tests and we probably shouldn't, but I agree to remove it. thx